README.md

```diff
@@ -54,22 +48,35 @@ Using this, you get this analysis FOR FREE (and it's fast!)
 
 - Branch x channel x VARIATION x experiment-id x PHASE (install, reject, alive etc) using UNIFIED TELEMETRY
 
+## <span id="engineering-hints">Engineering and Process</span>
 
-## Development
-
-- open an issue
-- hack and file a PR
+THIS IS TEMPORARY LIST, as of 2017-11-16.
 
+- [Shield article on Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield)
+- [Shield Studies article on Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield/Shield_Studies)
+- [mozilla/shield-studies-addon-template/](https://github.com/mozilla/shield-studies-addon-template/)
+- [Current work-in-progress docs and launch process](https://github.com/mozilla/shield-studies-addon-utils/issues/93)
+- [Long, rambling engineering docs](./docs/engineering.md)
+- Come to slack:  #shield
 
 ## Gotchas, Opinions, Side Effects, and Misfeatures
 
 1.  No handling of 'timers'.  No saved state at all (including the variation name), unless you handle it yourself.
 
 2.  No 'running' pings in v4 (yet).
 
-3.  User disable also uninstalls (and cleans up)
+3.  User disable also uninstalls (and cleans up).
 
+## Development on the Utils
+
+- open an issue
+- hack and file a PR
 
-[example]: examples/README.md
 
+## history of major versions
 
+- v4.x: (proposed)  additional functions for common cases
+- v4: First `.jsm` release.  Uses packet format for PACKET version 3.
+- v3: Attempt to formalize on `shield-study` PACKET version 3.  Jetpack based.  Prototype used for `raymak/page-reload`.  All work abandoned, and no formal npm release in this series.  Work done at `v3-shield-packet-format` branch.  LAST JETPACK (addon-sdk) RELEASE.
+s v2: Code refactor to es6 `class` with event models.  Added cli tooling.  Packet format is still arbitrary and per-study.  Jetpack based.  Last used in studies in Q2 2017.
```

Comment:
This looks funky. I think that leading "s" should be a dash, to keep w/ `<ol>` structure.